tc/core/mapping_options_cpp_printer.cc

```diff
@@ -32,34 +34,25 @@ MappingOptionsCppPrinter& MappingOptionsCppPrinter::printSchedulerOptions(
   return *this;
 }
 
-MappingOptionsCppPrinter& operator<<(
-    MappingOptionsCppPrinter& prn,
-    const std::string& str) {
-  prn.printString(str);
-  return prn;
-}
-
-MappingOptionsCppPrinter& operator<<(
-    MappingOptionsCppPrinter& prn,
+MappingOptionsCppPrinter& MappingOptionsCppPrinter::print(
```

Comment:
do we really need to return `*this` from `print`?

Reply:
This is for consistency with the other functions.
I am fine not returning but is there something special about ```print``` vs say ```printSchedulerOptions```?